src/Microsoft.IdentityModel.Logging/ITelemetryInstrumentation.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.IdentityModel.Logging
+{
+    internal interface ITelemetryInstrumentation
+    {
+        internal void LogOperationDuration(
+            TimeSpan operationDuration);
+
+        internal void LogOperationDuration(
+            TimeSpan operationDuration,
+            Exception exception);
+
+        internal void IncrementOperationCounter(
```

Comment:
This API will not work anymore as soon as we start tracking something else. I'd advise renaming the methods to indicate what exactly is being tracked or perhaps adding a parameter that describes the operation.